server/public/utils/utils.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package utils
+
+func Contains[T comparable](slice []T, item T) bool {
```

Comment:
Thank you.

Added a test code for a Contain function
Added description of the function.

Reply:
Thanks. Cases for just int and string would have been sufficient. But it's okay since you have already added the code.

Reply:
> Thanks. Cases for just int and string would have been sufficient. But it's okay since you have already added the code.

Thanks for all the advice during PR.